### PR TITLE
Set up repository

### PR DESCRIPTION
**Pull request by emilyliu7321**
**Description:**

- Adds .gitignore, codeowner files
- Adds pre commit, unit test, and integration test workflows
- Bump `elex-solver` version to <2 
- Adds PR template 
- Updated Github labels 

**Test steps:**
1. Unit test and integration test workflows pass

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Integration Test
+on: [push, pull_request]
+jobs:
+  test:
+    name: Run integration test
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        python-version: [3.8]
```
Comment (chriszs) [inline .github/workflows/integration.yml:10]: Should this happen on 3.7, since that's what data-importer is set to?

**Review by chriszs — APPROVED** [review]
Looks good. One question.

File: .github/workflows/integration.yml (hunk 1 above)
Comment (emilyliu7321) [reply .github/workflows/integration.yml:10]: I think the data importer version should get bumped up instead!

Comment (chriszs) [reply .github/workflows/integration.yml:10]: Would it make any sense to matrix both versions for now, e.g. 
```suggestion
        python-version: [3.8, 3.7]
```

Comment (chriszs) [reply .github/workflows/integration.yml:10]: (This way we cover ourselves from introducing breakage into importer while preparing the dependencies for the glorious future.)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: test
+on: [push, pull_request]
+jobs:
+  test:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        python-version: [3.8]
```
Comment (emilyliu7321) [inline .github/workflows/test.yml:10]:
```suggestion
        python-version: [3.8, 3.7]
```

**Review by chriszs — APPROVED** [review]

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: Integration Test
+on: [push, pull_request]
```
Comment (chriszs) [inline .github/workflows/integration.yml:2]: See other comment.

**Comment by chriszs** [comment]
This is great, thank you! But seems like it's running some tests twice:

Screen Shot 2022-09-19 at 5 28 15 PM

See comment for idea about that.

**Review by chriszs — APPROVED** [review]

**Pull request merged**